src/qrl/core/node.py

```diff
@@ -27,6 +27,13 @@ def __init__(self,
 
 
 class POW(ConsensusMechanism):
+    """
+    This is named POW because back then QRL was PoS and this was just a
+    alternate implementation of the node that used PoW.
+
+    Some functions are now unused/obsolete - go-qrl will clean this up. Most of
```

Comment:
```suggestion
    Some functions are now unused/obsolete - go-qrl will clean this up. Most of
```
No need to talk about what go-qrl will be doing on the description. Better to be concise and talking about what this class does.